pkg/utils/common/common.go

```diff
@@ -287,29 +287,10 @@ func GenOpenAPI(inst *cue.Instance) (b []byte, err error) {
 	return out.Bytes(), nil
 }
 
-// extractParameterDefinitionNodeFromInstance extracts the `#parameter` ast.Node from root instance, if failed fall back to `parameter` by LookUpDef
+// RefineParameterValue refines cue value to merely include `parameter` identifier
 // nolint:staticcheck
```

Comment:
do we need nolint now？

Reply:
Yes, `ResolveReference` is deprecated and we still need it in this func.